File: nro_dod_proste_dodawanie/rno_dod.r
f <- file("stdin")
open(f)

t <- as.integer(readLines(f, n=1))

for(i in 1:t) {
	n <- as.integer(readLines(f, n=1))
	tab <- as.integer(strsplit(readLines(f, n=1)," ")[[1]])
	write(sum(tab), stdout())
}
close(f)

File: nro_dod_proste_dodawanie/rno_dod.f90
Program rno_dod

    integer :: t
    integer :: n
    integer :: i,j,sum
    integer, dimension(:), allocatable :: tab

    read(*,*) t

    do i=1, t
        sum = 0
        read (*,*) n
        allocate(tab(n))
        read(*,*) (tab(j), j=1,n)
        do j= 1, n
            sum = sum + tab(j)
        end do
        deallocate(tab)
        write(*,*) sum
    end do
end Program rno_dod